pkg/api/handlers/libpod/pods.go

```diff
@@ -230,13 +230,31 @@ func PodRestart(w http.ResponseWriter, r *http.Request) {
 func PodPrune(w http.ResponseWriter, r *http.Request) {
 	var (
 		runtime = r.Context().Value("runtime").(*libpod.Runtime)
+		decoder = r.Context().Value("decoder").(*schema.Decoder)
+		reports []*entities.PodPruneReport
 	)
-	pruned, err := runtime.PrunePods()
+	query := struct {
+		Force bool `schema:"force"`
```

Comment:
This does not match the usage of Force for pruning containers or volumes, we should drop this. Force is client-side-only and removes the yes/no continue prompt.

Reply:
@baude @jwhonce I thought we agreed to add the force for prune pod. 
Should I go ahead had change this to the client-side only?

Reply:
Fixed the force flag. @mheon PTAL